### ISSUE TITLE
Gallery: Color contrast too low

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14474]
An a11y review found the following contract issues:
- [ ] Contrast of text in drawer should possibly be increased from 2.99 to 4.50 for smaller text or 3.00 for larger text
- [ ] Contrast of Hamburger menu icon in main Flutter Gallery home page should be increased from 1.11 to 3.00 or higher
- [ ] Upper left hand arrow back button contrast  in Contacts Profile should be increased from 1.94 to 3.0 or higher (issue also found in other sections of app as well)

**Comment by johnsonmh** [comment]
I've written https://github.com/flutter/flutter/pull/48752 to address the first and third issues. The second point is no longer an issue I believe, as the hamburger menu icon is currently passing contrast ratio guidelines.
